scripts/label_validator.sh

```diff
@@ -145,6 +145,12 @@ function runOnPr() {
                     jq -r 'if (.labels | length) > 0 then .labels[] | objects | .name else empty end')
 
   count_of_required_labels=$(grep -o -w -F -c "${supported_labels}" <<< "$present_labels" || true)
+
+  if [[ $present_labels == *"do-not-merge"* ]]; then
```

Comment:
are there any more labels preventing the merge?

Reply:
Yes I checked it. It matches all "do-not-merge" labels.